InventoryBuilder.py

```diff
@@ -196,11 +222,15 @@ def query_inventory_permanent(self):
             self.provide_hosts()
             self.provide_datastores()
             self.provide_vms()
+            self.provide_distributed_vswitches()
             self.provide_nsxt_adapter()
             self.provide_nsxt_mgmt_cluster()
             self.provide_nsxt_mgmt_nodes()
             self.provide_nsxt_mgmt_service()
             self.provide_nsxt_transport_nodes()
+            self.provide_nsxt_logical_switches()
+            self.provide_vcops_self_monitoring_objects()
```

Comment:
why is this called `vcops_self_monitoring_objects` but everywhere else you are refering to `vcops_objects`?